### ISSUE TITLE
Some docstrings fail with `LookupError: names` for signature-translated functions

### ISSUE THREAD

**Issue by krassowski** [rpy2/rpy2#903]
**Describe the issue or bug**

As described in https://github.com/rpy2/rpy2/issues/902#issuecomment-1186350479, 47 docstrings fail with `LookupError: names` for signature-translated functions in `base` package.

**To Reproduce**

```python
from rpy2.robjects.packages import importr

base = importr('base')
print(base.Sys_Date.__doc__)
```

**Expected behavior**
Docstring gets shown (for example https://www.rdocumentation.org/packages/base/versions/3.6.2/topics/Sys.time )

**Error**

```python
LookupError                               Traceback (most recent call last)
Input In [199], in <cell line: 4>()
      1 from rpy2.robjects.packages import importr
      3 base = importr('base')
----> 4 base.Sys_Date.__doc__

File rpy2/robjects/functions.py:55, in DocstringProperty.__get__(self, obj, objtype)
     53     return self.class_doc
     54 else:
---> 55     return self.fget(obj)

File py2/robjects/functions.py:230, in DocumentedSTFunction.__doc__(self)
    227 doc.append(description)
    229 fm = _formals_fixed(self)
--> 230 names = fm.do_slot('names')
    231 doc.append(self.__rname__+'(')
    232 for key, val in self._prm_translate.items():

File rpy2/rinterface_lib/conversion.py:45, in _cdata_res_to_rinterface.<locals>._(*args, **kwargs)
     44 def _(*args, **kwargs):
---> 45     cdata = function(*args, **kwargs)
     46     # TODO: test cdata is of the expected CType
     47     return _cdata_to_rinterface(cdata)

File rpy2/rinterface_lib/sexp.py:168, in Sexp.do_slot(self, name)
    166     name_r = rmemory.protect(openrlib.rlib.Rf_install(cchar))
    167     if not _rinterface._has_slot(self.__sexp__._cdata, name_r):
--> 168         raise LookupError(name)
    169     res = openrlib.rlib.R_do_slot(self.__sexp__._cdata, name_r)
    170 return res

LookupError: names
````

**Additional context**

Script for checking entire packages in https://github.com/rpy2/rpy2/issues/902#issuecomment-1186350479

**Comment by lgautier** [comment]
I am wrongly assuming that all R functions have formals. I should check for an R NULL here.

**Comment by lgautier** [comment]
My mistake. This seems to be an issue with functions without arguments.

**Comment by lgautier** [comment]
Fixed with PR #907.
